cheatsheets/Swift.rb

```diff
@@ -344,8 +368,8 @@ class MyClass : MyProtocol {
       name 'Error Handling'
       notes <<-'END'
         ``` swift
-        // ErrorType
-        enum MyError : ErrorType {
+        // Error
+        enum MyError : Error {
```

Comment:
I'd consider conforming to `LocalizedError` and implementing `var errorDescription: String?` Something like: 
```swift
enum MyError: LocalizedError {
    case Err1
    case Err2(errDesc: String)
    
    var errorDescription: String? {
    switch self {
        case .Err1: "Error 1 occurred"
        case let .Err2(errDesc): "Error 2 occurred: \(errDesc)"
        }
    }
}

Then update line 397 to:

```swift
print(error.localizedDescription)
```